app/lib/features/news/widgets/news_item/news_side_bar.dart

```diff
@@ -89,6 +92,21 @@ class NewsSideBar extends ConsumerWidget {
                   shrinkWrap: false,
                   centerTitle: true,
                   useCompactEmptyState: false,
+                  postCreateComment: () async {
+                    // when user create a comment, let's autosubscribe to comments
+                    await autosubscribe(
```

Comment:
why not placing it directly in create comment function ?

Reply:
because as of now, we don't support autosubscribing to anything but the boosts. in a later version this might be different.